CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.20.0 Release
+- Release Date: June 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
+
+#### Tablet Improvements
+ - Channel sidebar now remains open at a fixed width on tablet devices.
+ 
+#### iOS Keyboard Dismissal
+ - If the keyboard is open, swiping down past it now closes it.
+
+### Improvements
+ - Increased the double tap delay for post action buttons.
+ - Implemented assets for Adaptive icons.
+ - Users are now brought to the bottom of the channel when posting a message.
+
```

Comment:
lets add "Added support on iOS for IPv6 on LTE networks"

Reply:
Support for group constraints with 5.12 servers ?

Reply:
Also "Users can now execute actions while the keyboard is open"